grammar/mongo.g4

```diff
@@ -33,7 +33,7 @@ arguments:
 
 argument: literal | objectLiteral | arrayLiteral;
 
-objectLiteral: '{' propertyNameAndValueList? ','? '}';
+objectLiteral: '{' propertyNameAndValueList? '}';
```

Comment:
Don't think you want this change - this removes the ability to have trailing commas, right?